android/build.gradle

```diff
@@ -37,11 +40,16 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
 }
 
 dependencies {
     implementation 'androidx.work:work-runtime:2.7.1'
-    implementation 'androidx.annotation:annotation:1.3.0'
-    implementation 'androidx.core:core:1.7.0'
-    implementation 'androidx.fragment:fragment:1.4.1'
+    implementation 'androidx.annotation:annotation:1.4.0'
+    implementation 'androidx.core:core:1.9.0'
+    implementation 'androidx.fragment:fragment:1.5.2'
+    // just a test
+    compileOnly 'io.flutter:flutter_embedding_debug:1.0.0-ad3d868e0d174f5a8447b011bd714d18a5f2abec'
```

Comment:
is this needed?

Reply:
This was a test to provide a dependency so that I don't need to edit the code in two Android Studio instances, it makes trouble as I see so I'll revert this